client/src/components/ProfilePhoto/useStyles.ts

```diff
@@ -33,6 +33,24 @@ const useStyles = makeStyles((theme) => ({
     padding: theme.spacing(2),
     fontWeight: 700,
   },
+  form: {
+    width: '100%',
+  },
+
+  photoLabel: {
+    padding: theme.spacing(2),
+    fontWeight: 700,
+    color: 'red',
+    border: '0.5px red solid',
+    // bordeRadius: '10px !important' not working
```

Comment:
Please remove commented code (Also, I think it is not working because of a typo in the property name)

Reply:
Thanks Rajiv, it's typo fixed